packages/java/parser-jvm-plugin-transfertypes/src/test/resources/dev/hilla/parser/plugins/transfertypes/push/openapi.json

```diff
@@ -0,0 +1,145 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Vaadin Application",
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "http://localhost:8080/connect",
+      "description": "Vaadin Backend"
+    }
+  ],
+  "tags": [
+    {
+      "name": "OtherEndpoint"
+    },
+    {
+      "name": "PushTypeEndpoint"
+    }
+  ],
+  "paths": {
+    "/OtherEndpoint/getMessageFlux": {
+      "post": {
+        "tags": ["OtherEndpoint"],
+        "operationId": "OtherEndpoint_getMessageFlux_POST",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "count": {
+                    "type": "integer",
+                    "format": "int32"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "nullable": true,
+                  "items": {
+                    "type": "string",
+                    "nullable": true
+                  },
+                  "x-class-name": "reactor.core.publisher.Flux"
```

Comment:
Apparently this refers to the original class name before the replacement. Is this intentional or just a natural consequence of things?

Reply:
Yes, I need that to identify those methods. Maybe could be renamed to `x-original-class`? But in many cases it just re-emits the same class name as in the API (see the many changes to existing tests).

Reply:
The name sounds good to me.